napari/_qt/qt_event_loop.py

```diff
@@ -190,9 +191,16 @@ def get_app(
         # workers at shutdown.
         app.aboutToQuit.connect(wait_for_workers_to_quit)
 
-        # this will register all of our resources (icons) with Qt, so that they
-        # can be used in qss files and elsewhere.
-        _register_napari_resources()
+        # Setup search paths for currently installed themes.
+        for name in _themes:
+            QDir.addSearchPath(f'theme_{name}', str(_theme_path(name)))
```

Comment:
👀 ... this is the key line in this PR
https://doc.qt.io/qt-5/qdir.html#addSearchPath